features/example_groups/shared_context.feature

```diff
@@ -1,15 +1,23 @@
 Feature: shared context
 
-  Use `shared_context` to define a block that will be evaluated in the context
-  of example groups either explicitly, using `include_context`, or implicitly by
-  matching metadata.
+  Use `shared_context` to define a block that will be evaluated in the context of example groups either locally, using `include_context` in an example group, or globally using `config.include_context`.
```

Comment:
Yep.  If we put line breaks, it'll render with line breaks on relish even if that's not a good place for it.__